plane/source/docs/guide-tailsitter.rst

```diff
@@ -0,0 +1,152 @@
+.. _guide-tailsitter:
+
+=================
+Tailsitter Planes
+=================
+
+In ArduPilot tail-sitters are any VTOL aircraft type that rotates the
+fuselage (and autopilot) then moving between forward flight and hover.
+
+Despite the name, not all tails-sitters land on their tails. Some are
+"belly landers", where they lie down flat for landing to improve
+takeoff and landing stability in wind. Some may have an undercarriage
+for wheeled takeoff and others may have a stand or other landing aid.
+
+All tails-sitters are considered types of QuadPlanes in ArduPilot. You
+should start off by reading the :ref:`QuadPlane documentation
+<quadplane-support` before moving onto this tailsitter specific
+documentation.
+
+Vectored and non-Vectored
+=========================
+
+ArduPilot sub-divides tail-sitters into two broad categories:
+
+- vectored tailsitters can tilt their rotors independently of the
+  movement of the fuselage, giving them vectored thrust
+- non-vectored tailsitters have fixed rotor orientation relative to
+  the fuselage, and rely on large control surfaces for hover authority
+
+Tailsitter Config
+=================
+
+The key parameter to make a plane a tailsitter is to set
+Q_FRAME_CLASS=10. That tells the QuadPlane code to use the tailsitter
```

Comment:
This needs to link to http://ardupilot.org/plane/docs/parameters.html#q-frame-class-frame-class